src/screens/home/views/OverlayView.tsx

```diff
@@ -121,6 +122,22 @@ const ShareDiagnosisCode = ({i18n}: {i18n: I18n}) => {
   );
 };
 
+const AccessibleView = ({children}: {children: React.ReactNode}) => {
+  const [screenReaderEnabled, setScreenReaderEnabled] = useState(false);
+  const handleScreenReaderToggled = (screenReaderEnabled: any) => {
+    setScreenReaderEnabled(screenReaderEnabled);
+  };
+  AccessibilityInfo.addEventListener('screenReaderChanged', handleScreenReaderToggled);
```

Comment:
We need to use `useEffect` and clean up listener proper. I suggest to create AccessibilityProvider at root (App.tsx) and provide a hook to get `screenReaderEnabled`.